docs/tutorial_ptk.rst

```diff
@@ -0,0 +1,139 @@
+.. _tutorial_ptk:
+
+***********************************************
+Tutorial: ``prompt_toolkit`` custom keybindings
+***********************************************
+
+Are you really jonesing for some special keybindings? We can help you out with
+that. The first time is free and so is every other time!
+
+.. warning:: This tutorial will let you hook directly into the
```

Comment:
This appears to be producing an unadorned `<div>`, which will be hard to style. GitHub quirk or how RST works?